Add Support for Nethermind on OP Docs (#1551)
ethereum-optimism/docs: pages/operators/node-operators/configuration/execution-config.mdx

import { Callout } from 'nextra/components'
import { Tabs } from 'nextra/components'

# Execution layer configuration options

<Callout type="info">
You can configure your node using the command line options (also called flags). 
There are also sub-commands, which can be used to invoke functionality such as the console or blockchain import/export.
</Callout>

<Tabs items={['op-geth', 'Nethermind']}>
  <Tabs.Tab>
    ## op-geth Configuration

    This section lists all configuration options for `op-geth`. `op-geth` implements the Execution-Layer, with minimal changes for a secure Ethereum-equivalent application environment. 
    The following are options from [v1.101308.0](https://github.com/ethereum-optimism/op-geth/releases/tag/v1.101308.0)
  </Tabs.Tab>
  <Tabs.Tab>
    ## Nethermind Configuration

    Nethermind provides a simpler configuration approach for OP Stack networks. To run Nethermind on an OP Stack network, you can use the `--network` flag followed by the network name:

    ```bash
    # For OP Mainnet

>     nethermind --network op-mainnet

Reviewer: Nethermind doesn't have a `network` option but `-c, --config`:

```suggestion
    nethermind -c op-mainnet
```

    # For OP Sepolia

>     nethermind --network op-sepolia

Reviewer: Nethermind doesn't have a `network` option but `-c, --config`:
```suggestion
    nethermind -c op-mainnet
```

    ```

>     For detailed Nethermind configuration options, please refer to the [Nethermind documentation](https://docs.nethermind.io/nethermind/ethereum-client/configuration).

Reviewer: The link is still working, but deprecated:

```suggestion
    For detailed Nethermind configuration options, please refer to the [Nethermind documentation](https://docs.nethermind.io/fundamentals/configuration).
```
